Gemfile

```diff
@@ -2,3 +2,5 @@ source "https://rubygems.org"
 
 # Specify your gem's dependencies in sem.gemspec
 gemspec
+
+gem "semaphore_client", :source => "https://gem.fury.io/renderedtext/"
```

Comment:
Let's open source it (open source repo + push to rubygems) before merging.

This way, only people who have access to our gemfury acc can install the gem.